opennms-doc/guide-admin/src/asciidoc/text/performance-data-collection/collectors/snmp/SnmpCollector.adoc

```diff
@@ -13,7 +13,7 @@ Access to the _SNMP Agent_ is configured through the _SNMP configuration_ in the
 |===
 | Class Name          | `org.opennms.netmgt.collectd.SnmpCollector`
 | Package             | core
-| Supported on Minion | `Yes` (via the SNMP proxy)
+| Supported on Minion | `Yes`
```

Comment:
It does use the SNMP proxy, which is different then how other collectors operate.

Reply:
But it is not something like Net-SNMP proxy functionality, right?

Reply:
Nope, it's some internal proxy for the RPCs.

What's important to note here is that this collector behaves differently then others - in particular how the TTL is handled.

Reply:
The reason for this change is, user tried to set up SNMP data collection for a Minion and configured the SNMP proxy host using the Minions IP address in "Configure OpenNMS -> Configures SNMP Community Names by IP Address", which is in my point of view a little always, open to any suggestions.

Reply:
Do you think renaming from `(via the SNMP proxy)` to `(via internal RPC proxy)` is better?

Reply:
I guess less is more in this case - users don't really need to care *how* it works, just know that it is supported on Minion.